intro-HPC/glossary.tex

```diff
@@ -136,12 +136,12 @@
 \newglossaryentry{Shared-memory-system}
 {
   name={Shared memory system},
-  description={Computing system in which all of the processors share one global memory space. However, access times from a processor to different regions of memory are not necessarily uniform},
+  description={Computing system in which all of the processors share one global memory space. However, access times from a processor to different regions of memory are not necessarily uniform. This is called NUMA: Non-uniform memory access. Memory closer the the CPU your jobs starts on will generally be faster to acces then memory that is closer to a different CPU. You can pin processes to a certain CPU to ensure they always use the same memory},
```

Comment:
- s/closer the the CPU/closer to the CPU/
- s/CPU your jobs start on/CPU where your jobs are executed on/ (we pin them, so starting amounts to executing, which you also mention at the end :)
- s/acces/access/
- s/then/than/